README.md

```diff
@@ -68,6 +68,7 @@ work. If omitted, all events will be sent. For a list of events, refer to the se
 
 ### Online booking only
 * `package_id` - optional - the ID of a package. This will hide the package selection dropdown and skip this step.
+* `package_ids` - optional - array with multiple package IDs. This will only show the selected packages in the package selection dropdown.
```

Comment:
Consider merging the parameters `package_id` and `package_ids` (with a clearly defined preference, i.e. deprecate `package_id` and fill `package_ids` from that value iff `package_ids` is unset; throw an error if both are set)